tools/obscuroscan_v2/frontend/src/components/BatchDataGrid.vue

```diff
@@ -47,14 +48,9 @@ export default {
   setup() {
     const store = useBatchStore()
 
-    // Start polling when the component is mounted
+    // Reload batch data onMount
     onMounted(() => {
-      store.startPolling()
```

Comment:
I wonder if this has the desired effect.. It seems like the `RotatingBatchesItem` is also activating the `store.startPolling()`